library/Erfurt/Store/Adapter/Stardog.php

```diff
@@ -0,0 +1,726 @@
+            return 0;
+        }
+
+        foreach ($deletes['results']['bindings'] as $key => $triple) {
+            $deleteTriples .= isset($triple['s']) ? "<".$triple['s']['value']."> " : $subjectSpec . " "; 
+            $deleteTriples .= isset($triple['p']) ? "<".$triple['p']['value']."> " : $predicateSpec . " ";             
+            if ( isset($triple['o']) ) {
+                $objectSpec = $this->getObjectSpec($triple['o']);
+            }
+            $deleteTriples .= $objectSpec . "." . PHP_EOL;
+        }
+
+        $this->_apiClient->remove( $deleteTriples, $graphUri, Erfurt_Store_Adapter_Stardog_ApiClient::FORMAT_TURTLE);
+        
+        return count($deletes);
+    }
+
+    /**
+     * @see Erfurt_Store_Adapter_Interface
+     */
+    public function deleteMultipleStatements($graphUri, array $statementsArray)
+    {
+        $triples = $this->buildTripleArray($statementsArray);
+        $count = count($triples);
+
+        $this->_apiClient->remove( implode($triples), $graphUri, Erfurt_Store_Adapter_Stardog_ApiClient::FORMAT_TURTLE);
+    }
+
+    /**
+     * @see Erfurt_Store_Adapter_Interface
+     */
+    public function deleteModel($graphUri)
+    {
+        $this->_graphs = null;
+        return $this->_apiClient->query('DROP SILENT GRAPH <' . $graphUri . '>');;
+    }
+
+    /**
+     * @see Erfurt_Store_Adapter_Interface
+     * @todo test this, what needed for?
+     */
+    public function getBlankNodePrefix()
+    {
+        return self::BLANKNODE_PREFIX;
+    }
+
+    /**
+     * @see Erfurt_Store
+     */
+    public function getGraphsUsingResource($resourceUri) {
+        $graphs = array();
+
+        $res = $this->sparqlQuery('SELECT DISTINCT ?graph { GRAPH ?graph { <'.$resourceUri.'> ?p ?o . } }');        
+        if ($res) {
+            foreach ($res as $key => $value) {
+                $graphs[] = $value['graph'];
+            }
+        }
+
+        return $graphs;
+    }
+
+    /**
+     * @see Erfurt_Store_Adapter_Interface
+     * @todo Support Export
+     */
+    public function getSupportedExportFormats()
+    {
+    	return array();
+    }
+
+    /**
+     * @see Erfurt_Store_Adapter_Interface
+     * @todo implement
+     */
+    public function exportRdf($graphUri, $serializationType = 'xml', $filename = null)
+    {
+        throw new Erfurt_Store_Adapter_Exception('RDF export not implemented yet.');
+    }
+
+    /**
+     * Return supportet import formats, but we dont need to support any, 
+     * because \Erfurt_Store does all
+     *
+     * @see Erfurt_Store_Adapter_Interface
+     */
+    public function getSupportedImportFormats()
+    {
+        // dont support any (Erfurt_Store does all)
+        return array();
+    }
+
+    /**
+     * Import RDF, will be done by addMultipleStatements() from \Erfurt_Store
+     *
+     * @see Erfurt_Store_Adapter_Interface
+     */
+    public function importRdf($graphUri, $data, $type, $locator)
+    {
+        $app = Erfurt_App::getInstance();
```

Comment:
just throw an excpetion as in export method, since Store class will handle with import when getSupportedImportFormats() is empty